streaming/src/test/scala/org/apache/spark/streaming/UISeleniumSuite.scala

```diff
@@ -125,24 +125,37 @@ class UISeleniumSuite
 
         // Check batch tables
         val h4Text = findAll(cssSelector("h4")).map(_.text).toSeq
-        h4Text.exists(_.matches("Active Batches \\(\\d+\\)")) should be (true)
         h4Text.exists(_.matches("Completed Batches \\(last \\d+ out of \\d+\\)")) should be (true)
```

Comment:
Is this still flaky isn't it? What if it reaches this line before at least one batch finishes?

Reply:
It should be fine now. In 2nd attempt it should get at least 1 complete batch. As second attempt is after 500 ms and batches are of 100 ms only.